fdbserver/ServerCheckpoint.actor.cpp

```diff
@@ -0,0 +1,47 @@
+#include "fdbserver/ServerCheckpoint.actor.h"
+#include "fdbserver/RocksDBCheckpointUtils.actor.h"
```

Comment:
must be compiled only with `SSD_ROCKSDB_EXPERIMENTAL`

Reply:
same here.